tests/modules/nanopb/lib/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Already done in the application folder but should be safe from any directory
```

Comment:
exactly as you point out, this is already done in app folder, so even though it's safe, then I assume there must be some reason to do it here also.

Could you please comment why we want to also include it here, when already done elsewhere.

Reply:
It was done purely as a demonstration, I can remove it if you want.

Reply:
if there is a dedicated purpose, like being able to re-use the CMakeLists.txt, or similar, then it may make sense to keep.
But then it should have a comment describing such purpose.

If there is no such purpose, then I believe it's better to remove (as I notice you have already done)

Reply:
It was demonstrating a common `cmake` function, so it doesn't make sense to draw attention to this, so I removed it.